pymc/distributions/multivariate.py

```diff
@@ -1820,16 +1817,14 @@ class MatrixNormal(Continuous):
     constant, both the covariance and scaling could be learned as follows
     (see the docstring of `LKJCholeskyCov` for more information about this)
 
-    .. code:: python
+    .. code-block:: python
 
         # Setup data
-        true_colcov = np.array([[1.0, 0.5, 0.1],
-                                [0.5, 1.0, 0.2],
-                                [0.1, 0.2, 1.0]])
+        true_colcov = np.array([[1.0, 0.5, 0.1], [0.5, 1.0, 0.2], [0.1, 0.2, 1.0]])
```

Comment:
Add trailing comma so it formats multiline (more readable that way)

```suggestion
        true_colcov = np.array([[1.0, 0.5, 0.1], [0.5, 1.0, 0.2], [0.1, 0.2, 1.0],])
```